src/wrapt/_wrappers.c

```diff
@@ -1318,14 +1320,22 @@ static PyObject *WraptObjectProxy_round(
         return NULL;
 
     dict = PyModule_GetDict(module);
-    round = PyDict_GetItemString(dict, "round");
 
+    #if PY_VERSION_HEX >= 0x30d0000  /* Python >=3.13 */
```

Comment:
Instead of this version check, under free threading is it instead safe to use:

```
    // dict = PyModule_GetDict(module);
    // round = PyDict_GetItemString(dict, "round");

    round = PyObject_GetAttrString(builtins, "round");
```

That way same code across all versions.

Reply:
FWIW. There was a bug in the `__round__` hook implementation anyway as it didn't accept `ndigits` argument. I have fixed that and changed  to using `PyObject_GetAttrString` on assumption that is okay. Because have also made other changes for `static` vs `const`, then the only change required for PR may well just be adding:

```
     #ifdef Py_GIL_DISABLED
         PyUnstable_Module_SetGIL(module, Py_MOD_GIL_NOT_USED);
     #endif
```

Can you rebase your changes against head of `develop` branch and check again.

Reply:
ACK.

I will rebase.